daprdocs/content/en/dotnet-sdk-docs/dotnet-error-handling/_index.md

```diff
@@ -0,0 +1,7 @@
+﻿---
+type: docs
+title: "Error Handling in the Dapr .NET SDK"
+linkTitle: "Error handling"
+weight: 50000
```

Comment:
Could you please change this weight to 90000? It should ideally show up under the various Dapr clients, but before Dev Integrations in the docs.

Reply:
changed now